10_rest/n0npax/cmd/puppy-server/main_test.go

```diff
@@ -0,0 +1,166 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"os"
+	"testing"
+
+	"bou.ke/monkey"
+
+	puppy "github.com/anz-bank/go-course/10_rest/n0npax/pkg/puppy"
+	store "github.com/anz-bank/go-course/10_rest/n0npax/pkg/puppy/store"
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseArgs(t *testing.T) {
+
```

Comment:
unnecessary blank line.

Reply:
one line between `const` and `func`. I believe it's outdated. Marking manually as resolved.